README.md

```diff
@@ -75,14 +90,6 @@ import keras_cv
 > [!IMPORTANT]
 > Make sure to set the `KERAS_BACKEND` before import any Keras libraries, it
 > will be used to set up Keras when it is first imported.
-Until the Keras 3 release, KerasCV will use a preview of Keras 3 on PyPI named
-[keras-core](https://pypi.org/project/keras-core/).
-
-> [!IMPORTANT]
-> If you set `KERAS_BACKEND` variable, you should `import keras_core as keras`
-> instead of `import keras`. This is a temporary step until Keras 3 is out!
-To restore the default **Keras 2** behavior, `unset KERAS_BACKEND` before
-importing Keras and KerasCV.
 
 Once that configuration step is done, you can just import KerasCV and start
 using it on top of your backend of choice:
```

Comment:
below this line, let's remove all `from keras_cv.backend import keras` and replace with `import keras`

Reply:
also let's test our quickstart snippet on keras 3!

Reply:
done and added the colab link in description
- https://colab.sandbox.google.com/drive/1ZvZOLgGbqdWCTvipmPXntEnptixuFo5j#scrollTo=QEQth_GEtfQ3